plugins/completion/cmp/sources/default.nix

```diff
@@ -197,7 +197,7 @@ in
 {
```

Comment:
You need to remove the `{ pluginName = "codeium-nvim"; sourceName = "codium"; }` attrs from the list above.

Otherwise we're creating the enable/package option twice, once using `mkCmpSourcePlugin` and once using `mkNeovimPlugin`.

Reply:
Doh! That was the thing that threw me off the wrong path...

Reply:
Thank you very much!